src/php/class-admin-options.php

```diff
@@ -444,7 +444,7 @@ function lazyload_settings_page()	{ ?>
 	}
 
 	function lazyload_admin_js() {
-		wp_enqueue_script( 'lazyload_admin_js', LL_URL . 'public/js/admin.js', array('jquery', 'jquery-ui-tabs', 'wp-color-picker' ), LL_VERSION );
+		wp_enqueue_script( 'lazyload_admin_js', LL_URL . 'public/js/admin.js', array('jquery', /* 'jquery-ui-tabs', */ 'wp-color-picker' ), LL_VERSION );
```

Comment:
It's great to no longer have this dependency on jquery-ui-tabs!

Reply:
I just realized that the actual tab functionality of showing and hiding is broken because of this.